---
title: "Cancer Patient Genome Analysis"
description: "Understanding the patient's genome to create better therapies"
date: 2018-12-20
weight: 1
portfolio: ["research"]
header_transparent: true
thumbnail: "images/gen/projects/research.webp"
hero:
  background: "images/gen/home/home-1-large.webp"
  blend_mode: "overlay"
  theme: "primary"
work:
  heading: PanDrugs
  heading_label: "Tool highlight"
  brief: "PanDrugs is a bioinformatics platform to prioritize anticancer drug treatments according to individual genomic data."
  link: "https://www.pandrugs.org"
work2:
  heading: VulcanSpot
  heading_label: "Tool highlight"
  brief: "VulcanSpot integrates molecular profiling with gene essentiality screenings in cancer cell lines in order to identify gene dependencies."
  link: "http://vulcanspot.org"
---

## Towards better cancer therapies

Identifying the most appropiate therapies based on cancer genome data is a major challenge in personalized cancer medicine. Physicians and researchers face with long lists of tumor-specific genomic variants where most variants are either clinically “unactionable”, their biological role unknown or they are irrelevant for tumor biology. Our group is interested in the development of novel bioinformatics methods to interpret genomic alterations, evaluate their clinical relevance and drug feasibility and provide a prioritized evidence-based list of tailored anticancer therapies to facilitate clinical decision making.

Cancer precision medicine has achieved promising results in the clinical practice, although only a small fraction of patients present the particular biomarkers of drug sensitivity to take benefit of this approach. For that reason, our group is also interested in identifying nnovel druggable cancer dependencies in order to expand the current catalogue of targeted therapies for cancer treatment.

Another line of our work is the search for new applications of existing drugs through drug repositioning, a procedure to find new applications to already approved drugs by using computational methods. Our lab is interested in  drug-associated gene expression signatures coming from large pharmacogenomics projects, drug structural info and artificial intelligence and machine learning algorithms to infer new potential therapeutical applications, to predict cancer drug response and to infer synergistic combinations of antitumor drugs.